overrides/resources/deepmoblearning/patchouli_books/book/book.json

```diff
@@ -1,21 +0,0 @@
-{
```

Comment:
If the patchouli book configs are all being deleted, how will that impact players who have a copy of the mod's guidebook? The only way we have available to get the mod guidebook is from the quest reward, to my knowledge.

Reply:
DME comes with a recipe to craft the book.